packages/core/src/scss/components/checkout/customer/_customer.scss

```diff
@@ -41,6 +41,8 @@
     @include breakpoint("small") {
         width: grid-calc(7, $total-columns);
     }
+
+    margin-top: 20px;
```

Comment:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/88361607/204455368-527f5c0f-4a83-4a0c-bb82-6c8da9f26729.png">
It looks like the text fields are larger than the normal (non-stripe) form.
Also, CSS changes like this will apply to all interface with or without floating labels. Could you please provide a preview on those non-stripe forms. Thanks.

Reply:
Hi @bc-peng, thanks for review this PR.  

https://user-images.githubusercontent.com/67211919/205139101-4d7cae49-2be5-40e6-a0c3-ff09092db9f2.mov

Reply:
Hey @er-contreras could you add a screend shot or demo whit the css chagnes where we could see the billing field (non-stripe fields)

Reply:
Sure, 

![Uploading Screenshot 2022-12-01 at 17.03.20.jpg…]()

Reply:
This line and the one in `_forms.css` appear to be breaking current checkout design.
Please refer to the screenshots.
<img width="500" alt="Screen Shot 2022-12-02 at 10 23 17 pm" src="https://user-images.githubusercontent.com/88361607/205283430-3aa04ef6-548f-4018-9fca-f96c46e6b3bc.png">
<img width="500" alt="Screen Shot 2022-12-02 at 10 26 07 pm" src="https://user-images.githubusercontent.com/88361607/205283437-4747e796-d526-42d6-a63f-13539894429c.png">

cc @animesh1987